geocoding/clean.py

```diff
@@ -0,0 +1,30 @@
+import fire
+import pandas as pd
+
+
+def cleaner(excel_sheet):
+    """Will combine all the CSV's formed during the geocoding process
+    
+    Args:
+        excel_sheet (str): The excel sheet that was geocoded
+    """
+    temp = pd.ExcelFile(excel_sheet)
+    sheet_names = temp.sheet_names
+
+    maindf = pd.read_csv(f"Sheets/{sheet_names[0]}.csv")
+    maindf = maindf.drop("Geocoded", axis=1)
+
+    total = len(maindf)
+    for sheet_name in sheet_names[1:]:
```

Comment:
What is the reason for starting from 1 and not 0?

Reply:
Refactored this file and renamed to a more appropriate name. Added comments to explain.